erp/export/models.py

```diff
@@ -0,0 +1,72 @@
+from dataclasses import dataclass, fields
+from typing import Literal, Set, Optional
+
+
+@dataclass
+class EtalabModel:
+    id: str
+    transport_station_presence: bool = None
+    transport_information: str = None
+    stationnement_presence: bool = None
+    stationnement_pmr: bool = None
+    stationnement_ext_presence: bool = None
+    stationnement_ext_pmr: bool = None
+    cheminement_ext_presence: bool = None
+    cheminement_ext_terrain_accidente: bool = None
+    cheminement_ext_plain_pied: bool = None
+    cheminement_ext_ascenseur: bool = None
+    cheminement_ext_nombre_marches: int = None
+    cheminement_ext_reperage_marches: bool = None
+    cheminement_ext_main_courante: bool = None
+    cheminement_ext_rampe: Literal["aucune", "fixe", "amovible"] = None
+    cheminement_ext_pente: Literal["aucune", "légère", "importante"] = None
+    cheminement_ext_devers: Literal["aucun", "léger", "important"] = None
+    cheminement_ext_bande_guidage: bool = None
+    cheminement_ext_retrecissement: bool = None
+    entree_reperage: bool = None
+    entree_vitree: bool = None
+    entree_vitree_vitrophanie: bool = None
+    entree_plain_pied: bool = None
+    entree_ascenseur: bool = None
+    entree_marches: int = None
+    entree_marches_reperage: bool = None
+    entree_marches_main_courante: bool = None
+    entree_marches_rampe: Literal["aucune", "fixe", "amovible"] = None
+    entree_dispositif_appel: bool = None
+    entree_dispositif_appel_type: Set[
+        Literal["bouton", "sonnette", "interphone", "visiophone"]
+    ] = None
+    entree_balise_sonore: bool = None
+    entree_aide_humaine: bool = None
+    entree_largeur_mini: int = None
+    entree_pmr: bool = None
+    entree_pmr_informations: str = None
+    accueil_visibilite: bool = None
```

Comment:
I'm totally foreign to static typing in Python, but I'm surprised `bool` typed members are nullable by design, are you sure we don't need an `Option`, `Maybe` or equivalent here?

Reply:
I want to force the explicit declaration of values, which can be None.

Reply:
So it's not a `bool`, but an `Optional[bool]`, right?